src/Jumbotron.js

```diff
@@ -2,11 +2,21 @@ import React from 'react';
 import classNames from 'classnames';
 
 const Jumbotron = React.createClass({
+  propTypes: {
+    componentClass: React.PropTypes.node.isRequired
```

Comment:
Ok, I’ve changed it to `any`. A more complete solution would still make sense, but I think it is out of the scope of this PR, because it should be fixed for all other components with a `componetClass` prop too.